tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/signup_info.h

```diff
@@ -58,11 +56,16 @@ class SignupInfo {
 SignupInfo* deserialize_signup_info(
     const std::string& serialized_signup_info);
 
-std::map<std::string, std::string> CreateEnclaveData(
-    const std::string& originator_public_key_hash);
+tcf_err_t DeserializePublicEnclaveData(
+    const std::string& public_enclave_data,
+    std::string& verifying_key,
+    std::string& encryption_key,
+    std::string& encryption_key_signature);
```

Comment:
Is this function `DeserializePublicEnclaveData()`. exposed via swig interface . If not, you may want to keep this function be inside signupInfo class? Only those functions that needs to be called by python (via swig) and which don't dependent on a class need to be declared as global functions

Reply:
DeserializePublicEnclaveData() function is not exposed via swig interface, but its used in other files like signup_info_kme.cpp and signup_info_wpe.cpp to deserialize public enclave data.
When we have class for each of these files, then it can be addressed.